src/sentry_plugins/twilio/plugin.py

```diff
@@ -126,6 +122,15 @@ def get_send_to(self, *args, **kwargs):
         # This doesn't depend on email permission... stuff.
         return True
 
+    def error_message_from_json(self, data):
+        code = data.get("code")
```

Comment:
@ceorourke Is `code` different than the status code?

Reply:
yes, `code` in this case is a Twilio error code: https://www.twilio.com/docs/api/errors